.github/workflows/pr.yaml

```
@@ -0,0 +1,23 @@
+name: Docker Image from PR
+
+on:
+  pull_request:
+    branches:
+      - master
+    types:
+      - assigned
+      - edited
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - uses: olegtarasov/get-tag@v1
+      id: tagName
+    - name: Build the Docker images
+      run: |
+        export GITHUB_TAG_NAME=${GITHUB_REF##*/}
```

```suggestion
        export GITHUB_TAG_NAME=${GITHUB_HEAD_REF##*/}
```

Otherwise it will use `merge` as a tag name.

---

Done